spec/index.bs

```diff
@@ -6558,23 +6558,25 @@ pipeline, and have the following members:
                 [=Device timeline=] |initialization steps|:
 
                 1. Let |limits| be |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
+                1. Let |bindGroupLayouts| be a copy of |descriptor|.{{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
+                1. For each |i| in the [=list/get the indices|indices=] of |bindGroupLayouts|:
+                    1. If |bindGroupLayouts|[|i|] is `undefined` or [=list/empty=], set |bindGroupLayouts|[|i|] to `null`.
```

Comment:
@kainino0x I've just realized `|bindGroupLayouts|[|i|] is [=list/empty=]` is actually not defined in the spec as https://gpuweb.github.io/gpuweb/#gpubindgrouplayout is not a list per say. We obviously mean empty is a bind group layout with 0 entries but we may want to actually define this. WDYT?

Reply:
Ah good catch, yes, we should say something like ``If |bindGroupLayouts|[|i|] is `undefined` or |bindGroupLayouts|[|i|].{{GPUBindGroupLayout/[[descriptor]]}}.{{GPUBindGroupLayoutDescriptor/entries}} is [=list/empty=]`

Reply:
Here's https://github.com/gpuweb/gpuweb/pull/4958